aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

```diff
@@ -382,18 +382,15 @@ The JSON Schema library maps standard C# types to OpenAPI `type` and `format` as
 | object         | _omitted_      |                  |
 | dynamic        | _omitted_      |                  |
 
-Note that object and dynamic types have _no_ type defined in the OpenAPI because
-these can contain data of any type, including primitive types like int or string.
+Note that object and dynamic types have _no_ type defined in the OpenAPI because these can contain data of any type, including primitive types like int or string.
 
-The `type` and `format` can also be set with a [Schema Transformer]. For example,
-you may want the `format` of decimal types to be `decimal` instead of `double`.
+The `type` and `format` can also be set with a [Schema Transformer]. For example, you may want the `format` of decimal types to be `decimal` instead of `double`.
```

Comment:
```suggestion
The `type` and `format` can also be set with a [Schema Transformer](#use-schema-transformers). For example, you may want the `format` of decimal types to be `decimal` instead of `double`.
```